utils/docker/Dockerfile.ubuntu.20.04

```
@@ -68,10 +55,15 @@ COPY utils/sl utils/sl
 COPY utils/build.config utils/
 RUN chown -R daos_server.daos_server /home/daos
 USER daos_server:daos_server
-RUN ln -s utils/sl site_scons
 
-RUN [ "$DAOS_DEPS_BUILD" != "yes" ] || \
-    (scons --build-deps=yes install PREFIX=/opt/daos --deps-only && \
+# Control what to build.  By default Dockerfiles build everything to allow for
+# ease-of-use for users, however in CI everything is turned off and then
+# selectively enabled.  Turning off any step disables all future steps.
+ARG DAOS_DEPS_BUILD=yes
+ARG DEPS_JOBS=1
+
+RUN ln -s utils/sl site_scons && [ "$DAOS_DEPS_BUILD" != "yes" ] || \
```

Use the more natural boolean logic available in the Bourne shell, and avoid double negatives:
```suggestion
RUN ln -s utils/sl site_scons && "$DAOS_DEPS_BUILD" && \
```